src/commands/logout.rs

```diff
@@ -1,14 +1,24 @@
+use std::path::PathBuf;
 use structopt::StructOpt;
```

Comment:
```suggestion
use std::path::PathBuf;

use structopt::StructOpt;
```

Let's group `std` imports away from external crates. rust-analyzer picks up on this, and it helps reading when things are sorted!